app.js

```diff
@@ -0,0 +1,109 @@
+let board = ['', '', '', '', '', '', '', '', ''];
+
+function endGameColor() {
+  const e = document.getElementsByClassName('square');
+  for (let i = 0; i < e.length; i += 1) {
+    e[i].style.backgroundColor = 'rgb(14, 68, 3)';
+  }
+}
+function normalGameColor() {
```

Comment:
[OPTIONAL]
Since you are using ES6 syntax in other parts, consider using it here as well